documentation/config_file_auth.md

```diff
@@ -159,10 +185,71 @@ Puppet Server allows agents to make requests at the old URLs and internally tran
 
 ## Related Configuration Settings
 
-> **Deprecation Note:** The `auth.conf` rules for the master endpoints, and client whitelists for the Puppet admin and certificate status endpoints, are deprecated. Convert your configuration files to the HOCON formats and configure the following settings to allow only the new authorization method.
-
 For backward compatibility, settings in [`puppetserver.conf`][] also control whether to use the new Puppet Server authorization method for certain endpoints:
 
 * `use-legacy-auth-conf` in the `jruby-puppet` section: If `true`, Puppet Server uses the Ruby authorization methods and  [Puppet `auth.conf`][] format and warns you that this is deprecated. If `false`, Puppet Server uses the new authorization method and HOCON `auth.conf` format. Default: `true`.
 * `authorization-required` and `client-whitelist` in the `puppet-admin` section: If `authorization-required` is set to `false` or `client-whitelist` has at least one entry, Puppet Server authorizes requests to Puppet Server's administrative API according to the parameters' values. See the [`puppetserver.conf` documentation][`puppetserver.conf`] for more information on these settings. If `authorization-required` is set to `true` or not set and `client-whitelist` is set to an empty list or not set, Puppet Server authorizes requests to Puppet Server's administrative API using the authorization method introduced in Puppet Server 2.2.
 * `certificate-status.authorization-required` and `certificate-status.client-whitelist` in the `certificate-authority` section: If `authorization-required` is set to `false` or `client-whitelist` has one or more entries, Puppet Server handles requests made to its [Certificate Status](/puppet/latest/reference/http_api/http_certificate_status.html) API according to the parameters' values. See the [`ca.conf` documentation](./config_file_ca.html) for more information on these settings. If `authorization-required` is set to `true` or not set and the `client-whitelist` is set to an empty list or not set, Puppet Server authorizes requests using the authorization method introduced in Puppet Server 2.2.
+
+## Deprecated Ruby Parameters
```

Comment:
I'd want that in a different page. This one's long enough as a reference to both rule formats, IIRC there are other considerations in migrating between formats, and a separate migration document would be easier to remove when the older format is removed.